### PR TITLE
Rephrase "Creating Custom Fragments" Section

### LINKED ISSUE #2087: Rephrase "Creating Custom Fragments" section
### Please confirm that you have searched existing issues in the repo

Yes, I have searched the existing issues

### Any related issues?

_No response_

### What is the area that this feature belongs to?

_No response_

### Is your feature request related to a problem? Please describe.

https://markbind.org/userGuide/reusingContents.html#creating-custom-fragments

This section is positioned and phrased somewhat oddly within the page, it also sounds like an extra feature but it is not.

### Describe the solution you'd like

Let's communicate clearly with simpler terms, and provide sufficient context of the site config file.

Some suggestions
- Rename the heading to "Excluding Files from Rendering as Pages", or anything similar, clearer and more concise. Edit the content accordingly.
- Move it under above the "Reusing Contents Across Sites" section, under "Includes", as it is also useful with just includes, even without subsites.
- Provide a reason for using this (creating files that are only included)
- Include a link for directing the reader to the actual feature (`pagesExclude`) in the site configuration file with more details.
- Anything else, feel free to modify the above suggestions as well.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by itsyme**
**What is the purpose of this pull request?**

- [x] Documentation update
- [ ] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] DevOps
- [ ] Improve developer experience
- [ ] Others, please explain:

**Overview of changes:**
Resolves #2087. Edited the "Creating Custom Fragments" based on suggestions in the issue and more. Added a tooltip for "custom fragments" to make description look cleaner.

**Anything you'd like to highlight/discuss:**
While addressing this issue, I realised that the inline code is not being recognised for the warning box in ["Creating Custom Fragments"](https://markbind.org/userGuide/reusingContents.html#creating-custom-fragments). I will investigate this and open an issue for it.

Edit: I realised this was caused by a missing empty line after the `<box>` and fixed it accordingly.

**Testing instructions:**

**Proposed commit message: (wrap lines at 72 characters)**
Rephrase Creating Custom Fragments section

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [x] Linked all related issues
- [x] No unrelated changes

### CODE REVIEW

File: docs/userGuide/reusingContents.md [hunk 1]
```diff
@@ -59,12 +59,9 @@ If you are using Git for version control, you can set up the sub-site repository
...
 </include>
 
+#### Excluding Files from Rendering as Pages
 
-<hr><!-- ======================================================================================================= -->
-
-## Creating Custom Fragments
-
-**MarkBind supports creating custom fragments**. A fragment is a piece of content that can be reused across multiple pages. This allows you to create reusable content similar to [reusing content](#reusing-contents-across-sites), but rather than reusing content from a _sub-site_, content is reused from fragments which can be excluded from page generation.
+**MarkBind supports the exlusion of files from rendering as pages**. This allows for the ease of creation of <tooltip content="A fragment is a piece of content that can be reused across multiple pages.">custom fragments</tooltip>. This allows you to create reusable content similar to [reusing content](#reusing-contents-across-sites), but rather than reusing content from a _sub-site_, content is reused from fragments which are excluded from page generation.
```
Comment (lhw-1) [inline docs/userGuide/reusingContents.md:64]: I think "exclusion" is misspelled as "exlusion" in the bolded sentence!

Comment (lhw-1) [inline docs/userGuide/reusingContents.md:64]: I think since the "Reusing Contents Across Sites" section is now the parent section for this section, perhaps simply referring to it could be an alternative to linking it?

**Comment by lhw-1** [comment]
Left some comments!

Also, as part of the original issue, I think maybe we can also address these points:

- Provide a reason for using this (creating files that are only included)
- Include a link for directing the reader to the actual feature (pagesExclude) in the site configuration file with more details.

**Comment by itsyme** [comment]
Hi @lhw-1! Thanks for the comments! I have made the improvements in the latest commit based on some of your suggestions!

File: docs/userGuide/reusingContents.md (hunk 1 above)
Comment (itsyme) [reply docs/userGuide/reusingContents.md]: Good catch! Thank you!

Comment (itsyme) [reply docs/userGuide/reusingContents.md]: I agree! I think it will look cleaner as well!

File: docs/userGuide/reusingContents.md [hunk 2]
```diff
@@ -59,14 +59,12 @@ If you are using Git for version control, you can set up the sub-site repository
...
 </include>
 
+#### Excluding Files from Rendering as Pages
 
-<hr><!-- ======================================================================================================= -->
-
-## Creating Custom Fragments
-
-**MarkBind supports creating custom fragments**. A fragment is a piece of content that can be reused across multiple pages. This allows you to create reusable content similar to [reusing content](#reusing-contents-across-sites), but rather than reusing content from a _sub-site_, content is reused from fragments which can be excluded from page generation.
+**MarkBind supports the exclusion of files from rendering as pages**. This allows for the ease of creation of <tooltip content="A fragment is a piece of content that can be reused across multiple pages.">custom fragments</tooltip>. This allows you to create reusable content similar to reusing content, but rather than reusing content from a _sub-site_, content is reused from fragments which are excluded from page generation. This facilitates the use of repeated content across the site.
```
Comment (tlylt) [inline docs/userGuide/reusingContents.md:64]: @itsyme Could we perhaps rephrase this further to make it clearer? It seems a little lengthy and the link to "reusing content" has been removed, which makes this part confusing:
> This allows you to create reusable content similar to reusing content, but rather than reusing content from a _sub-site_, content is reused from fragments which are excluded from page generation.

wdyt?

Comment (itsyme) [reply docs/userGuide/reusingContents.md:64]: I agree about the lengthiness! I will rephrase it to keep it more succinct. As for the removal of the link, I initially agreed with what @lhw-1 said above, the reasoning being that this section is already under the reusing content section. However, I would be open to inserting the link back as well to make it seem clearer.

File: docs/userGuide/reusingContents.md [hunk 3]
```diff
@@ -59,14 +59,12 @@ If you are using Git for version control, you can set up the sub-site repository
...
 </include>
 
+#### Excluding Files from Rendering as Pages
 
-<hr><!-- ======================================================================================================= -->
-
-## Creating Custom Fragments
-
-**MarkBind supports creating custom fragments**. A fragment is a piece of content that can be reused across multiple pages. This allows you to create reusable content similar to [reusing content](#reusing-contents-across-sites), but rather than reusing content from a _sub-site_, content is reused from fragments which can be excluded from page generation.
+**MarkBind supports the exclusion of files from rendering as pages**. This allows for the ease of creation of <tooltip content="A fragment is a piece of content that can be reused across multiple pages.">custom fragments</tooltip>. This allows you to create reusable content similar to [reusing content](#reusing-contents-across-sites), but instead of a _sub-site_, content is reused from fragments which are excluded from page generation. This facilitates using repeated content across pages.
```
Comment (tlylt) [inline docs/userGuide/reusingContents.md:64]:
```suggestion
**MarkBind supports the exclusion of files from rendering as pages**. For example, you can exclude files containing <tooltip content="A fragment is a piece of content that can be reused across multiple pages.">custom fragments</tooltip> of reusable content from page generation.
```

Comment (tlylt) [reply docs/userGuide/reusingContents.md:64]: I think there's a typo in the original issue, which says that this part should:
> Move it ~~under~~ above the "Reusing Contents Across Sites" section, under "Includes", as it is also useful with just includes, even without subsites.

**Review by tlylt — CHANGES_REQUESTED** [review]

File: docs/userGuide/reusingContents.md (hunk 3 above)
Comment (itsyme) [reply docs/userGuide/reusingContents.md]: Oh I see, I think this makes more sense as well! I have made the relevant changes in the latest commit!

File: docs/userGuide/syntax/includes.md [hunk 4]
```diff
@@ -156,6 +156,68 @@ You can also specify include variables within the `<include>` tag itself by addi
 
 If the same variable is defined in a chain of `<include>`s (e.g. `a.md` includes `b.md` includes `c.md`...), variables defined in the top-most `<include>` will take precedence. Global variables (`_markbind/variables.md`) will take precedence over any `<include>` variables.
 
+<hr><!-- ======================================================================================================= -->
+
+##### Excluding Files from Rendering as Pages
+
+**MarkBind supports the exclusion of files from rendering as pages**. For example, you can exclude files containing <tooltip content="A fragment is a piece of content that can be reused across multiple pages.">custom fragments</tooltip> of reusable content from page generation
```
Comment (tlylt) [inline docs/userGuide/syntax/includes.md:163]:
```suggestion
**MarkBind supports the exclusion of files from rendering as pages**. For example, you can exclude files containing <tooltip content="A fragment is a piece of content that can be reused across multiple pages.">custom fragments</tooltip> of reusable content from page generation.
```

**Review by tlylt — APPROVED** [review]
